File: sqlparser/src/sql/SqlMetaQueryStatement.java
//@
package cdata.sql;
//@
/*#
using RSSBus.core;
using RSSBus;
namespace CData.Sql {
#*/

import core.ParserCore;

public class SqlMetaQueryStatement extends SqlStatement {
  private String Query;

  public SqlMetaQueryStatement(String text, Dialect dialectProcessor) {
    super(dialectProcessor);
    this.Query = text;
  }

  public SqlMetaQueryStatement(SqlTokenizer tokenizer, Dialect dialectProcessor) throws Exception {
    super(dialectProcessor);
    tokenizer.NextToken();
    this.Query = tokenizer.NextToken().Value;
    ParserCore.parseComment(this, tokenizer);
  }

  private SqlMetaQueryStatement(Dialect dialectProcessor) {
    super(dialectProcessor);
  }

  public String getQuery() {
    return this.Query;
  }

  public /*#override#*/ void accept(ISqlQueryVisitor visitor) throws Exception {

  }

  @Override
  public Object clone() {
    SqlMetaQueryStatement obj = new SqlMetaQueryStatement(null);
    obj.copy(this);
    return obj;
  }

  @Override
  protected void copy(SqlStatement obj) {
    super.copy(obj);
    SqlMetaQueryStatement o = (SqlMetaQueryStatement)obj;
    this.Query = o.Query;
  }
}

File: sqlparser/src/sql/csgen/SqlMetaQueryStatement.cs
using System;
using System.Collections;
using System.IO;
using System.Threading;
using System.Net;
using System.Net.Sockets;
using System.Text;
using RSSBus.core.j2cs;



using RSSBus.core;
using RSSBus;
namespace CData.Sql {


public class SqlMetaQueryStatement : SqlStatement {
  private string Query;

  public SqlMetaQueryStatement(string text, Dialect dialectProcessor) : base(dialectProcessor) {
    this.Query = text;
  }

  public SqlMetaQueryStatement(SqlTokenizer tokenizer, Dialect dialectProcessor) : base(dialectProcessor) {
    tokenizer.NextToken();
    this.Query = tokenizer.NextToken().Value;
    ParserCore.ParseComment(this, tokenizer);
  }

  private SqlMetaQueryStatement(Dialect dialectProcessor) : base(dialectProcessor) {
  }

  public string GetQuery() {
    return this.Query;
  }

  public override void Accept(ISqlQueryVisitor visitor) {

  }

  public override Object Clone() {
    SqlMetaQueryStatement obj = new SqlMetaQueryStatement(null);
    obj.Copy(this);
    return obj;
  }

  protected override void Copy(SqlStatement obj) {
    base.Copy(obj);
    SqlMetaQueryStatement o = (SqlMetaQueryStatement)obj;
    this.Query = o.Query;
  }
}
}

